apps/client/sttr_build/parser.js

```diff
@@ -1,5 +1,28 @@
-const parser = require("fast-xml-parser");
-const get = require("lodash.get");
+import { DomHandler } from "https://deno.land/x/domhandler/mod.ts";
+import { Parser } from "https://rawcdn.githack.com/tbjgolden/deno-htmlparser2/5522f6286a17cc3857c5f1aa30e59e82968de822/htmlparser2/index.ts";
+
+const ENABLE_LOGGING = false;
+
```

Comment:
Is this working already? Had a issue with generate un parser.js

Reply:
Yes it works when the parser is updated.